src/contracts/core/AVSDirectory.sol

```diff
@@ -413,6 +413,24 @@ contract AVSDirectory is
         }
     }
 
+    /// @inheritdoc IAVSDirectory
+    function isOperatorSetStrategy(OperatorSet calldata operatorSet, IStrategy strategy) external view returns (bool) {
+        return _operatorSetStrategies[_encodeOperatorSet(operatorSet)].contains(address(strategy));
+    }
+
+    /// @inheritdoc IAVSDirectory
+    function isOperatorSetStrategyBatch(
+        OperatorSet calldata operatorSet,
+        IStrategy[] calldata strategies
+    ) external view returns (bool) {
+        for (uint256 i; i < strategies.length; ++i) {
+            if (!_operatorSetStrategies[_encodeOperatorSet(operatorSet)].contains(address(strategies[i]))) {
```

Comment:
Can we add tests to the AVSD to validate this is calculated correctly? Just sanity checking the output from those fuzz tests